forge-gui/res/cardsfolder/c/combat_research.txt

```diff
@@ -2,7 +2,7 @@ Name:Combat Research
 ManaCost:U
 Types:Enchantment Aura
 K:Enchant creature
-A:SP$ Attach | ValidTgts$ Creature | AILogic$ Curiosity | AITgts$ Creature.Legendary
+A:SP$ Attach | ValidTgts$ Creature | AILogic$ Curiosity | AITgts$ Creature.Legendary+YouCtrl
```

Comment:
I don't see why these changes are needed
The AILogic already selects its own creatures

Reply:
The AITgts for Combat Research currently causes it to prefer a legendary creature an opponent controls over a nonlegendary creature the AI controls (I had a game where an effect suspended Combat Research and, once the last time counter was removed, the AI controlled only nonlegendary creatures and it targeted an opponent's legend). That being said, I have removed the added AITgts for the other three cards

Reply:
ah right, thanks